spec/javascript/showStopTest.js

```diff
@@ -35,12 +62,50 @@ describe('show page test', () => {
       expect(component.find('.id')).toBePresent()
       expect(component.find('.name')).toBePresent()
       expect(component.state('stop')).toEqual({
-        id:1,
-        name:"El Greco",
-        created_at:"2019-04-22T18:53:17.323Z",
-        updated_at:"2019-04-22T18:53:17.323Z"
+        "id": 1,
+        "name": "Edward Hopper",
+        "created_at": "2019-04-23T16:43:36.475Z",
+        "updated_at": "2019-04-23T16:43:36.475Z"
       });
-      expect(component.find('.name').text()).toEqual("Name: El Greco")
+      expect(component.find('.name').text()).toEqual("Name: Edward Hopper")
+      done()
+    },0)
+  });
+
+  it('should show review data', (done) => {
+    setTimeout(() => {
+      expect(component.find('.reviews')).toBePresent()
+      expect(component.state('reviews')).toEqual(
+        [
+          {
+            "id": 1,
+            "rating": 2,
+            "body": "not enough pasta, too much sauce",
+            "stop_id": 1,
+            "created_at": "2019-04-24T19:20:15.921Z",
+            "updated_at": "2019-04-24T19:20:15.921Z"
+          },
+          {
+            "id": 3,
+            "rating": 5,
+            "body": "Best stop ever",
+            "stop_id": 1,
+            "created_at": "2019-04-24T20:44:18.854Z",
+            "updated_at": "2019-04-24T20:44:18.854Z"
+          },
+          {
+            "id": 4,
+            "rating": 3,
+            "body": "Don't stop here",
+            "stop_id": 1,
+            "created_at": "2019-04-24T20:45:34.178Z",
+            "updated_at": "2019-04-24T20:45:34.178Z"
+          }
+        ]
+      );
+
+      expect(component.find('.rating').first().text()).toEqual('Rating: 2')
+      expect(component.find('.review_body').first().text()).toEqual('Body: not enough pasta, too much sauce')
       done()
```

Comment:
you should also be testing that your Review component appears on the page, and that it is given the correct props.

Doing so would also replace the tests on line 107 and 108

Reply:
Removing the tests on line 107 and 108 is the only note we didn't address. I'm happy to take these out but I'd like to make the argument that without them we are not testing that the review component is actually displaying any data, only mounting and receiving data.

Reply:
Per our conversation I'm good with your approach.